src/main.py

```diff
@@ -524,31 +532,22 @@ def loop():
 				player.play_speech(resources_path + 'alexayes.mp3')
 				time.sleep(0.5)
```

Comment:
What is this delay for after the "yes?" I feel like it's an unnecessary pause before starting recording.